packages/uswds-core/src/styles/mixins/helpers/checkbox-and-radio-colors.scss

```diff
@@ -118,6 +131,41 @@
 
   @if $input-type == "checkbox" {
     .usa-checkbox__input {
+      // Unchecked radio buttons receive :indeterminate pseudo-class.
+      // Because of this, we need to explicetly target checkbox :indeterminate styles.
+      &:indeterminate,
+      &[data-indeterminate] {
+        @include format-input {
+          @include add-background-svg($input-indeterminate);
+          @include selected-input-colors($input-active-color);
+          background-position: center center;
+          background-size: units(1.5) auto;
+
+          @media (forced-colors: active) {
+            @include add-background-svg("checkbox-indeterminate-alt");
+            background-color: color(SelectedItem);
+          }
+        }
+
+        &:disabled,
+        &[aria-disabled="true"] {
+          @include format-input {
+            box-shadow: $tile-box-shadow--disabled;
+          }
+
+          @include format-label {
+            border-color: $tile-border-color--disabled;
+          }
+        }
+      }
```

Comment:
## Disabled conflict fix

Moving the checkbox:indeterminate styles caused the newer `format-input` styles to take priority. These styles are required to maintain disabled box shadow and tile colors.

Alternatively, we can move these styles to the disabled styles defined in lines 182-191. I opted to move them here since they weren't required to be redefined for the disabled checkboxes without `indeterminate`.